index.html

```diff
@@ -303,9 +302,9 @@
       <p>The `PerformanceObserver(callback)` constructor must create a new
       <a>PerformanceObserver</a> object with <a>PerformanceObserverCallback</a>
       set to <var>callback</var> and then return it.</p>
-      <p>A <dfn>registered performance observer</dfn> consists of an observer
-      (a <a>PerformanceObserver</a> object) and options (a
-      <a>PerformanceObserverInit</a> dictionary).</p>
+      <p>A <dfn>registered performance observer</dfn> is a <a>struct</a>
+      consisting of an <var>observer</var> (a <a>PerformanceObserver</a> object)
+      and <var>options</var> (a <a>PerformanceObserverInit</a> dictionary).</p>
```

Comment:
The options does not exit until observe is called, so this seems confusing,

Reply:
Yep, and we have the plumbing in place in observe. Here we're simply defining what a "registered performance observer" is.